common/cosmwasm-smart-contracts/vesting-contract/src/messages.rs

```diff
@@ -113,6 +113,9 @@ pub enum ExecuteMsg {
     UpdateStakingAddress {
         to_address: Option<String>,
     },
+    UpdateLockedPledgeCap {
+        amount: Uint128,
```

Comment:
This is more of a general thought rather than comment to this specific line of code, but recently I've been thinking about how we're using `Uint128`/`u128` to represent coin values in various places. I think in general we should be using the `Coin` type directly instead wherever possible as to be explicit about the denomination and not get accidentally screwed if somebody decides to do something weird with coins of different denom of if we accidentally messed up a migration or something.

Reply:
I agree, it’s just that denominations are a pain, if we had(maybe have) good constructors for Coin I’d be using them more.

Reply:
I'd say `pub fn new(amount: u128, denom: impl Into<String>)` is pretty good : )